hotel-reservation-system/src/App.js

```diff
@@ -1,30 +1,16 @@
 import { React, useEffect } from "react";
 import NavBar from "./components/shared/NavBar";
-import Login from "./components/Authorization/Login";
-import Register from "./components/Authorization/Register";
-import Home from "./components/Home";
 import API from "./api";
-import {
-  BrowserRouter as Router,
-  Switch,
-  Route,
-  Redirect,
-} from "react-router-dom";
-import { useDispatch, useSelector } from "react-redux";
-import {
-  IS_LOGGED,
-  NAME,
-  ROLE,
-  EMAIL,
-  USER_ID,
-} from "./storage/actions/actionTypes";
-import HotelTable from "./components/Hotel/HotelTable";
-import HotelEditor from "./components/Hotel/HotelEditor";
-import RoomsPage from "./components/Room/RoomsPage";
+import { useHistory } from "react-router";
+import { BrowserRouter as Router } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import RouteList from "./Routing/RouteList";
 import createAuthRefreshInterceptor from "axios-auth-refresh";
+import { FillStorage, Logout } from "./components/Authorization/TokenData";
```

Comment:
Please follow the next imports rule 
1. Libs
2. Absolute paths
3. Relative paths

Reply:
all imports are reorganized